lib/jenkins.rb

```diff
@@ -0,0 +1,74 @@
+require 'net/https'
+require 'thread'
+
+module BushSlicer
+  class Jenkins
+    attr_accessor :client, :build_map
+    def initialize
+      @client = JenkinsApi::Client.new(:server_url => 'https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com',
```

Comment:
please do not hardcode this. it needs to be a config under `private/config/config.yaml`. Actually it is already there under ```
services:
  jenkins: ...
```

Reply:
@akostadinov  how do I get the `conf`